extension/src/background_scripts/background.js

```diff
@@ -37,6 +39,14 @@ async function handleAnswerQuestionIncorrectly(request, reply) {
     reply(result);
 }
 
+async function handleFetchCurrentStudentInfo(request, reply) {
+    let result = await fetchCurrentStudentInfo(
+        request.studentName
```

Comment:
we don't need this anymore, `request` too

Reply:
Why do we not need this?

Reply:
in `fetchCurrentStudentInfo`, you are not using it. Consequently, `request` becomes an unused variable

Reply:
Right. Done